### ISSUE TITLE
Markgroups not rendering correctly using setup

### ISSUE THREAD

**Issue by Tempus14** [sdaps/sdaps#254]
Using markgroups and marklines, the setup routine breakes. The latex compilation stops after the fist introduction of a markline and no pdf will be generated. I am not shure whether this is a 'sdaps' or 'sdaps-class' problem since it will also occur when starting pdflatex standalone from the same folder (even though I can then ignore the errors in interactive mode and get an output - but still the markline question text ist rendered as 'ff'), but it will work perfectly fine using Texmaker ( for example with pdflatex as selected engine). So I guess that this is not a installation issue.

**Comment by benzea** [comment]
Can you please attach the failing document and ideally the log too?

**Comment by Tempus14** [comment]
Sure.
[github-export.zip](https://github.com/sdaps/sdaps/files/12240616/github-export.zip)
I uploaded the working directory in zip format since github does not support .tex format.

**Comment by Tempus14** [comment]
[questionnaire.pdf](https://github.com/sdaps/sdaps/files/12240679/questionnaire.pdf)
This is what the output should look like. (I also wanted to attach a broken compile while calling pdflatex from the commandline, but could not reproduce my previous fail. So this working render is also produced using the pdflatex command without any additional arguments.)

**Comment by Tempus14** [comment]
Great thanks for this hint. I will try that and report back. 
What version of python are you using then? It says 3.6 is neccessary and I was therefore using a VM running Ubuntu 18.04 as it build with 3.6 in mind. Turns out this is the reason I was running an old version of sdaps as it was collecting version 1.9.10 for Ubuntu 18.04. I would therefore upgrade this VM anyway and wanted to ask if 3.6 is still important or if I could get away with using a newer version of python.

**Comment by benzea** [comment]
Oh, python 3.6 is just the *minimum* version that is required.

I think there was some change recently in python and older versions stopped working with python 3.10 or so, but the current version should be compatible with all python releases >= 3.6. Though, who knows, I haven't tested 3.6 for ages …

**Comment by Tempus14** [comment]
Great. I tried that and also learned another lesson. When initally installing sdaps, I couldn't get it working on my main machine. Judging from the error messages I thought it was a problem with my python installation. So I settled for the virtual machine with python 3.6 and it worked. So this was not a big deal since I use sdaps only once a year or so. (This is also the reason why I was not checking for updates regularly, since the VM is also only started up more or less once a year.)
Comming back now years later and wiser (xD) I tried it again and turns out my inital problem back then was not about the python installation itself but about the conda enviroment. 
So now everything works fine and runs on my main sytem. This should make working with sdaps way nicer :D. Thank you very much for your help and your real quick response times. I really like working with sdaps (aside from installing it). Thank you very much.
